src/solvers/strings/array_pool.cpp

```diff
@@ -7,6 +7,8 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 \*******************************************************************/
 
 #include "array_pool.h"
+#include "string_refinement_util.h"
```

Comment:
:pick: ` etc.` in the commit message is less than ideal, perhaps it should be multiple commits, but should almost certainly list all the changes